Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -43,9 +43,11 @@
 
 #include "VideoCommon/OnScreenDisplay.h"
 
-// The minimum time it takes for the DVD drive to process a command (in
-// microseconds)
-constexpr u64 COMMAND_LATENCY_US = 300;
+// The minimum time it takes for the DVD drive to process a command (in microseconds)
+constexpr u64 MINIMUM_COMMAND_LATENCY_US = 300;
```

Comment:
I get that the `_US`, is supposed to stand for microseconds (µs), but this is redundant, since it's already explained in the comment. Like it is, it can be confused for "United States", as if this latency is specific to US consoles. I recommend that part of the name to be omitted, reading `MINIMUM_COMMAND_LATENCY`

Reply:
It's only redundant if you're looking at this specific line. The suffix is useful when you see the variable name referenced in the code, as it explains why we're dividing by 1000000.

Reply:
Got it!